src/main/java/managers/JSExecutor.java

```diff
@@ -15,6 +15,19 @@ public JSExecutor(WebDriver driver) {
         js = (JavascriptExecutor) driver;
     }
 
+    public boolean waitForJQueryToLoad() {
+        try {
+            return (Long) executeScript("return jQuery.active") == 0;
+        } catch (Exception e) {
```

Comment:
I'm missing the wait part. This code is only checking if jQuery is active once. How do you plan to use this method?

Reply:
Obviously you're right, just fixed that